TraineeTasks/StringTasks.cs

```diff
@@ -2,69 +2,26 @@
 using System.Collections.Generic;
 using System.Linq;
 
-namespace FirstPartTasks
+namespace TraineeTasks
 {
-    public class TwoChairsTasks
+    public class StringTasks
     {
-        public string ChangeEvenBitsToZero(int number)
-        {
-            if (number < 0)
-            {
-                return "The number must be greater than zero";
-            }
-
-            var binary = Convert.ToString(number, 2);
-            string result = "";
-
-            for (int i = 0; i < binary.Length; i++)
-            {
-                result = (i % 2) != 0 ? result += 0 : result += binary[i];
-            }
-
-            return result;
-        }
-
-        public int FindCountBitsEqualOne(int number)
-        {
-            if (number < 0)
-            {
-                return -1;
-            }
-
-            var binary = Convert.ToString(number, 2);
-
-            return binary.Count(c => c == '1');
-        }
-
         public string ChangeLetterCase(string str)
         {
-            if (String.IsNullOrEmpty(str))
+            if (string.IsNullOrWhiteSpace(str))
             {
-                return "The string must not be Null or Empty";
-            }
-
-            var result = "";
-
-            for (int i = 0; i < str.Length; i++)
-            {
-                if (str[i].ToString().ToLower() == str[i].ToString())
-                {
-                    result += str[i].ToString().ToUpper();
-                }
-                else
-                {
-                    result += str[i].ToString().ToLower();
-                }
+                throw new ArgumentException("The string cannot be null or empty string");
             }
 
-            return result;
+            return new String(str.Select(ch => char.ToLower(ch) == ch ?
+                char.ToUpper(ch) : char.ToLower(ch)).ToArray());
```

Comment:
Please define a private method for expression that inside a 'Select'
And than use like this:

```
var aaa = str.Select(ReverseCharCase);

return string.Concat(aaa);
```

Reply:
done